docs/rulebook.md

```diff
@@ -0,0 +1,100 @@
+# Rulebook / constitution
+
+This is a small document that defines how this community is working.
+
+## Project role
+
+Provide a shared development environment and infrastructure for the Nix and
+NixOS ecosystem, strictly for non-profit purposes.
+
+## Members
+
+There are currently two classes of members in the project:
+
+* Community members - participants of our community.
+* Administrators - members who own and administer the project.
+
+## Community members
+
+### Team
+
+Anybody who contributes code to one of our project is considered a community
+member.
+
+### Responsibilities
+
+Community members are asked to help keep a good spirit focused on building and
+shipping software. Everybody is human and shall be treated with compassion and
+respect.
+
+### Power
+
+> FIXME: this is not available yet.
+
+Community members might create a referendum. Any referendum that has more than
+10 votes gets submitted to the community at large for voting. The voting
+result will be enacted on a majority vote.
+
+## Administrators
+
+### Team
+
+* @Mic92
+* @adisbladis
```

Comment:
Is @adisbladis still actively doing stuff here? There is also winter, who would is maintaning the darwin build box: https://github.com/winterqt/darwin-build-box/commits/main

Reply:
Not a lot, but member rotation can happen separately. This PR is about introducing the mechanisms.

Reply:
Not too much on this repo any more. I'd still like to be an org admin regardless as I do way more in that role than I do in this repo.